en-US/Grammar.xml

```diff
@@ -624,7 +624,7 @@
 							</thead>
 							 <tbody>
 								<row>
-									<entry> Once the file retrieval has been completed, you are ready to restart your system. </entry>
+									<entry> After the file retrieval has been completed, you are ready to restart your system. </entry>
```

Comment:
I would leave this instance of "once" in place because it demonstrates a "before and after" scenario. The "before" uses "once" poor language; the "after" removes "once" and improves the language.

Reply:
Made requested change.